core/tests/protractor_utils/forms.js

```diff
@@ -497,6 +538,9 @@ var RichTextChecker = async function(arrayOfElems, arrayOfTexts, fullText) {
   var justPassedRteComponent = false;
 
   var _readFormattedText = async function(text, tagName) {
+    await waitFor.visibilityOf(
+      await arrayOfElems.get(
+        arrayPointer), `element at ${arrayPointer} is not visible`);
```

Comment:
I think this line break pattern is more readable

```suggestion
      await arrayOfElems.get(arrayPointer),
      `element at ${arrayPointer} is not visible`);
```